### DISCUSSION TITLE
Kimai 2.0.x - very slow reactions of the UI to input

### DISCUSSION THREAD

**Discussion by VOOM108** [kimai/kimai#4033] [Questions & Answers]
This seems to have become a little better with 2.0.21, but I still have the issue, that the reaction to any kind of input is very slow. 

Meaning: I type, and the characters lag behind significantly. Same with scrolling through menu items, changing the focus to another form etc. 

This is only the case inside modals, though! If not in a modal, everything is fine...

**Comment by kevinpapst** [comment]
Interesting. Never heard of that. 
Do you have excessive long lists of customers/projects/activities?
Can you try with a fresh browser profile in private mode (to make sure that there are no plugins causing that).
Sounds like a JS problem if it only happens in modals. 
Which Browser and OS? Anything in the javascript console?

**Comment by VOOM108** [comment]
There are no excessive lists... 

In Vivaldi there is indeed a javascript issue in the console - every single character throws an error like this: 
![image](https://github.com/kimai/kimai/assets/16562946/72f70e4e-9197-4ad1-9797-6f8819a37aed)

Using Firefox, where only plugin is active, the error is not present. Also: the dark mode works there. But it used to work in Vivaldi earlier, too. 

I now also tried Chrome, with almost the same plugins as Vivaldi - no problem there, too. Both input and dark mode are fine. 

Vivaldi has many features, that I really like. But it has also many shortcomings, other Chromium-based browsers don't. 

For the timekeeping, I will create a "webapp" out of Chrome now instead of Vivaldi.

**Reply by kevinpapst** [reply]
Hm, that's a pitty. content.js is nothing from Kimai, so this is indeed some incompatibility.
I use Brave, which works pretty well.
But Kimai is also working good as "standalone" Chrome app. 

So, the browser switch fixed both issues. Good to read 👍
